src/main/java/meteordevelopment/meteorclient/gui/widgets/input/WDoubleEdit.java

```diff
@@ -107,7 +108,7 @@ public double get() {
     }
 
     public void set(double value) {
-        this.value = value;
+        this.value = Utils.clamp(value, min, max);
```

Comment:
Here `min` and `max` are boxed Double types and not primitives, that means they can be null and null means infinite. Same gues for int edit.